diff --git a/node_modules/electron-installer-redhat/resources/spec.ejs b/node_modules/electron-installer-redhat/resources/spec.ejs
index e07a8a3..fec6f48 100644
--- a/node_modules/electron-installer-redhat/resources/spec.ejs
+++ b/node_modules/electron-installer-redhat/resources/spec.ejs
@@ -1,8 +1,6 @@
 %define _binary_payload w<%= compressionLevel %>.xzdio
 
-%if "%{_host_cpu}" != "%{_target_cpu}"
 %global __strip /bin/true
-%endif
 
 Name: <%= name %>
 Version: <%= version %>
@@ -24,7 +22,7 @@ print('\n\n\n') }
 
 %>%install
 mkdir -p %{buildroot}/usr/
-cp <%= process.platform === 'darwin' ? '-R' : '-r' %> usr/* %{buildroot}/usr/
+if [ -d usr ]; then SRC=usr; else SRC=../usr; fi && cp <%= process.platform === 'darwin' ? '-R' : '-r' %> $SRC/* %{buildroot}/usr/
 
 
 %files
